index.html

```diff
@@ -45,26 +46,26 @@ <h1>I’m Godfrey<br>
         coded, don’t hestiate to contact me.</p>
       <a href="">LET’S CONNECT</a>
       <ul>
-        <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt=""></a></li>
-        <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt=""></a></li>
-        <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt=""></a></li>
-        <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt=""></a></li>
-        <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt=""></a></li>
+        <li class="item1"><a href=""><img src="./image/Vectorgithub.svg" alt="github icon"></a></li>
+        <li class="item1"><a href=""><img src="./image/Linkedin iconlinkedin.svg" alt="linkedin icon"></a></li>
+        <li class="item1"><a href=""><img src="./image/Vectormedium.svg" alt="medium icon"></a></li>
+        <li class="item1"><a href=""><img src="./image/Vectortwitter (1).svg" alt="twitter icon"></a></li>
+        <li class="item1"><a href=""><img src="./image/Vector (3).svg" alt="unknown icon"></a></li>
       </ul>
     </div>
   
   </section>
-  <section class="work">
+  <section class="work" id="">
     <div class="container_work">
```

Comment:
- [ ] According to the project requirement, clicking on the menu items should take you to a specific part of the page.
> When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- Currently, the portfolio menu item does not take you to this portfolio section, please you need to fix this as its part of the project requirements. Thanks :+1: :wave: 
- SCREENSHOT-
![Selection_008](https://user-images.githubusercontent.com/107632780/232631975-152fe938-dfe4-4084-b1f8-45c0a249b799.png)


- [OPTIONAL] You can add this to the top of your `style.css` file to give it a smooth scroll effect which gives a great user experience `html {
  scroll-behavior: smooth;
}`